packages/terra-i18n/src/I18nProvider.jsx

```diff
@@ -18,13 +18,26 @@ const propTypes = {
   messages: PropTypes.object.isRequired,
 };
 
-const I18nProvider = props => (
-  <IntlProvider locale={props.locale} key={props.locale} messages={props.messages}>
-    <div>
-      {props.children}
-    </div>
-  </IntlProvider>
-);
+const I18nProvider = (props) => {
+  const {
+    locale,
+    messages,
+    ...customProps
+  } = props;
+
+  return (
+    <IntlProvider
+      {...customProps}
+      locale={props.locale}
+      key={props.locale}
+      messages={props.messages}
```

Comment:
nit: Could just use the const `locale` and `messages` instead of `props.locale` and `props.messages`.